ui.apps/src/main/content/jcr_root/apps/wknd/components/pagesearch/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
```

Comment:
the dialog should be cleaned up. The thing we've spoken about (additional tabs) are here because of presence of <asset> and <metadata> elements along with slingResourceSuperType used on compnent level. This means that the asset and metadata dialog elements are fetched from parent's resources.

This dialog should consists of a few required elements and yours desired name & maxnumberofresults.
![image](https://github.com/jarrecky/aem-tutorial/assets/33866638/de4fc2c9-60c0-4936-a499-1829a8d9dcbe)

Reply:
try to put <columns> right after <content>